R/pkg/inst/tests/testthat/test_context.R

```diff
@@ -24,7 +24,7 @@ test_that("Check masked functions", {
   func <- lapply(masked, function(x) { capture.output(showMethods(x))[[1]] })
   funcSparkROrEmpty <- grepl("\\(package SparkR\\)$|^$", func)
   maskedBySparkR <- masked[funcSparkROrEmpty]
-  namesOfMasked <- c("describe", "cov", "filter", "lag", "na.omit", "predict", "sd", "var",
+  namesOfMasked <- c("read.csv", "write.csv", "describe", "cov", "filter", "lag", "na.omit", "predict", "sd", "var",
```

Comment:
@felixcheung @sun-rui (Firstly, sorry I am not pretty much fluent to R).

I am not too sure why it says they are being masked. I anyway added those here and could pass all tests but still I am wondering why it is and if it is correct or not. When I change the function name to another, it looks fine but it looks there are no such functions to mask.

It looks it masks the functions called `read.csv` and `write.csv`. The console output was below:

```
The following object is masked from ‘package:testthat’:

    describe

The following objects are masked from ‘package:stats’:

    cov, filter, lag, na.omit, predict, sd, var

The following objects are masked from ‘package:utils’:

    read.csv, write.csv

The following objects are masked from ‘package:base’:

    colnames, colnames<-, drop, intersect, rank, rbind, sample, subset,
    summary, transform
```

Would you please give me some advices on this?